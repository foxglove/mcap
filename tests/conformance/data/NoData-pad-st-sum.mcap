version https://git-lfs.github.com/spec/v1
oid sha256:d06da22ab74f9a9ccf3b027a15402529c782930f8dd45e76492ccbba4a431b63
size 233
