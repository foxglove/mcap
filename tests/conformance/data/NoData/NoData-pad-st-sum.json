{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {"type": "DataEnd", "fields": [["data_section_crc", "150695454"]]},
    {
      "type": "Statistics",
      "fields": [
        ["attachment_count", "0"],
        ["channel_count", "0"],
        ["channel_message_counts", {}],
        ["chunk_count", "0"],
        ["message_count", "0"],
        ["message_end_time", "0"],
        ["message_start_time", "0"],
        ["metadata_count", "0"],
        ["schema_count", "0"]
      ]
    },
    {
      "type": "SummaryOffset",
      "fields": [
        ["group_length", "58"],
        ["group_opcode", "11"],
        ["group_start", "44"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "3614834834"],
        ["summary_offset_start", "102"],
        ["summary_start", "44"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["st", "sum", "pad"]}}
}
