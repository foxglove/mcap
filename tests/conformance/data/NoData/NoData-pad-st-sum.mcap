version https://git-lfs.github.com/spec/v1
oid sha256:73f93df5dfef0b042761c28f173b5a3c258a23e84a7b50266857b2d53d3c966d
size 204
