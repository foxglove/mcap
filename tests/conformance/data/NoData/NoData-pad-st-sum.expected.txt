Header library=s:"" profile=s:""
Statistics attachment_count=i:0 channel_count=i:0 channel_message_counts=m:{} chunk_count=i:0 message_count=n:0
SummaryOffset group_length=n:0 group_opcode=i:13 group_start=n:80
SummaryOffset group_length=n:36 group_opcode=i:11 group_start=n:44
SummaryOffset group_length=n:0 group_opcode=i:8 group_start=n:80
Footer summary_crc=i:0 summary_offset_start=n:80 summary_start=n:44
